plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -1383,7 +1373,10 @@ public String buildInsertSql(JdbcOutputTableHandle handle, List<WriteFunction> c
         checkArgument(handle.getColumnNames().size() == columnWriters.size(), "handle and columnWriters mismatch: %s, %s", handle, columnWriters);
         return format(
                 "INSERT INTO %s (%s%s) VALUES (%s%s)",
-                quoted(handle.getCatalogName(), handle.getSchemaName(), handle.getTemporaryTableName().orElseGet(handle::getTableName)),
+                quoted(
```

Comment:
We need a `quoted(RemoteTableName)` function. It is very common thing to quote it.

Reply:
There already exists the function. Here the `temporaryTableName` go first, so can not use it.